pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -292,6 +324,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: EditRoleName,
+				Annotations: map[string]string{
+					oapi.OpenShiftDescription: "A user that can modify most objects in a project, but does not have the power to view or modify roles or bindings."},
```

Comment:
A user that can create and edit most objects in a project, but can not update the project's membership.